src/commands/package.js

```diff
@@ -59,10 +58,19 @@ export default asyncCommand({
 		let builddir = path.resolve(cwd, argv.builddir || 'build');
 		let dest = path.resolve(cwd, argv.dest || 'pkg', `${argv.name}.zip`);
 
+		// Get label and description from src/intl/en_US.json or from argument
+		const { label, description } = getZimletLabelDescription() || argv;
+
+		if (!(label && description)) {
+			return error(`label and description string are not provided as argument`);
+		}
+
+		createLocalizationPropertyFiles(argv.name);
+
 		// Create the xml descriptor file for the zimlet
 		let xmlFile = `${argv.name}.xml`;
 
-		let zimletXML = `<zimlet name="${argv.name}" version="${argv.pkgVersion}" description="${argv.description}" label="${argv.label}" zimbraXZimletCompatibleSemVer="${argv.zimbraXVersion}">`;
+		let zimletXML = `<zimlet name="${argv.name}" version="${argv.pkgVersion}" description="${description}" label="${label}" zimbraXZimletCompatibleSemVer="${argv.zimbraXVersion}">`;
```

Comment:
are we using values of description and label here directly?

Reply:
We are destructuring both value  at https://github.com/Zimbra/zimlet-cli/pull/139/files#diff-94dd905335c883b52a8b7729b9d5f4568d4c71f06f1b600e326b4b1b4ed0bbcbR62